Orange/widgets/data/tests/test_oweditdomain.py

```diff
@@ -169,6 +170,33 @@ def test_duplicate_names(self):
         output = self.get_output(self.widget.Outputs.data)
         self.assertIsInstance(output, Table)
 
+    def test_time_variable_preservation(self):
+        """Test if time variables preserve format specific attributes"""
+        table = Table("cyber-security-breaches")
+        self.send_signal(self.widget.Inputs.data, table)
+        output = self.get_output(self.widget.Outputs.data)
+        self.assertEqual(str(table[0, 4]), str(output[0, 4]))
+
+        # if continuous variable edit is used have_time and have_date is not
+        # copied and the time string format is changed
+        idx = self.widget.domain_view.model().index(4)
+        self.widget.domain_view.setCurrentIndex(idx)
+        editor = self.widget.editor_stack.findChild(ContinuousVariableEditor)
```

Comment:
Not sure why this is tested her. Can ContinuousVariableEditor be used for editing TimeVariable by users?